lib/ui/landing_page.dart

```diff
@@ -82,28 +99,14 @@ class _LandingPageState extends State<LandingPage> {
       ),
     );
   }
-  // Gets the App Title and returns it
-  Widget _getAppTitleWidget() {
-    return new Text(
-      'Barber Sharp',
-      style: new TextStyle(
-        color: Colors.white,
-        fontWeight: FontWeight.bold,
```

Comment:
There are cases when we need a custom app bar, but this is not one of them in my opinion. I think the standard app bar will work fine.